src/main/java/net/fabricmc/loader/launch/FabricTweaker.java

```diff
@@ -122,6 +122,8 @@ public void injectIntoClassLoader(LaunchClassLoader launchClassLoader) {
 			}
 		}
 
+		FabricLoader.INSTANCE.getAccessWidener().loadFromMods(FabricLoader.INSTANCE);
```

Comment:
This seems a bit redundant to me to call a method on the loader's instance, and then require the loader again as a param.

Couldn't we just make the loader one of the ctor params for the AccessWideners since we won't usually be passing multiple instances of loader into the same game.

Reply:
Done.